demo/lib/vm.bash

```diff
@@ -30,6 +30,10 @@ vms:
         grep -E -v '^ *$'
 }
 
+vm-bootstrap-commands() {
```

Comment:
Before this patch the `distro-bootstrap-commands` function only prints commands to be executed when bootstrapping the system. Usually these are executed by cloud-init. I hesitate a bit making it a dual-operation function: printing or executing.

Do you think we could keep it only a printing function? Maybe new `vm-bootstrap` could store the bootstrap commands printed by `distro-bootstrap-commands` on vm and runs them as root? I mean like:
```
vm-bootstrap() {
    distro-bootstrap-commands | vm-pipe-to-file "./e2e-bootstrap.sh"
    vm-command "sh ./e2e-bootstrap.sh"
}
```

(Sure this needs revisiting the bootstrap commands so that they would not add duplicate lines to the same .conf files every time executed.)

Reply:
Sure, that can be done like it. I was not sure why those commands were not immediately executed so did it like that.